README.md

```diff
@@ -12,12 +12,60 @@ A user-friendly IRC bouncer.
 
 ## Usage
 
-    go run ./cmd/sojuctl create-user <username> -admin
+See the man page at `doc/soju.1.scd` for more information.
+
+### Server side
+
+    go run ./cmd/sojuctl create-user <soju username> -admin
     go run ./cmd/soju -listen irc+insecure://127.0.0.1:6667
```

Comment:
For people who are running the bouncer on another machine (like a home server or VPS), using 127.0.0.1 as the IP won't allow remote connections. A better choice here is to use 0.0.0.0 I believe.